Bulls_Cows.py

```diff
@@ -0,0 +1,70 @@
+import random
+import time
+
+
+def generate_digits():
+    digits = []
+    while len(digits) < 4:
+        random_digit = random.randint(0,9)
+        if random_digit not in digits:
+            digits.append(random_digit)
+    return digits
+
+
+def ask_number():
+    print("Enter a number")
+    num = input()
```

Comment:
Why don't you use `input("Enter a number ")`? If you want it to have the input on the new line you can use `input("Enter a number\n")

Don't abbreviate variable names. This should be `number`